packages/schematics/angular/workspace/files/tslint.json

```diff
@@ -61,15 +61,26 @@
       "single"
     ],
     "trailing-comma": false,
-    "no-output-on-prefix": true,
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "banana-in-box": true,
+    "component-class-suffix": true,
+    "contextual-life-cycle": true,
+    "directive-class-suffix": true,
+    "enforce-component-selector": true,
+    "no-attribute-parameter-decorator": true,
+    "no-conflicting-life-cycle-hooks": true,
     "no-input-rename": true,
+    "no-output-named-after-standard-event": true,
+    "no-output-on-prefix": true,
     "no-output-rename": true,
+    "no-queries-parameter": true,
+    "pipe-impure": true,
+    "prefer-inline-decorator": true,
+    "prefer-output-readonly": true,
```

Comment:
Any opinions about this one? Makes sense to me, requires a bit more typing though.

Reply:
I don't have a strong opinion on this one. Let me know what the team decides, and I'll update the PR accordingly.

Reply:
You can drop it, yes.